src/basic-syntax/doc-comments.md

```diff
@@ -0,0 +1,27 @@
+# Documentation Comments
```

Comment:
Rustdoc is the name I've usually heard used for this, where are they called Documentation Comments?

It would be good to link to https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html too.

Reply:
BTW these are referred to as doc comments in the language reference - https://doc.rust-lang.org/reference/comments.html